src/Components/Input.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * This returns an input field of a given type.
+ *
+ * @param {Object} props - The props object for the InputField component.
+ * @param {string} props.type - The type of the input field (e.g., "text", "password").
+ * @param {string} props.placeholder - The placeholder text for the input field.
+ * @param {function} [props.onBlur] - The event handler function for the input field's onBlur event.
+ * @param {string} props.value - The current value of the input field.
+ * @param {function} props.Onchange - The event handler function for the input field's onChange event.
+ * @returns {JSX.Element} A styled input field.
+ */
+function InputField({ type, placeholder, onBlur, value, onchange }) {
```

Comment:
```suggestion
function InputField({ type, placeholder, onBlur, value, onChange }) {
```
You've missed the case here.

Reply:
Fixed 340f5fed709c391b0a9add10ce1b0b50af9fb758